src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/court/ApplicationInsightsEventListener.kt

```diff
@@ -19,16 +19,16 @@ class ApplicationInsightsEventListener(
 ) : VideoLinkBookingEventListener {
   override fun bookingCreated(
     booking: VideoLinkBooking,
-    specification: VideoLinkBookingSpecification,
-    agencyId: String
+    specification: VideoLinkBookingSpecification
+
   ) {
     val properties = mutableMapOf(
       "id" to (booking.id?.toString()),
       "bookingId" to booking.offenderBookingId.toString(),
       "court" to specification.court,
       "courtId" to specification.courtId,
       "user" to authenticationFacade.currentUsername,
-      "agencyId" to agencyId,
+      "prisonId" to booking.prisonId,
```

Comment:
we shouldn't change the name of this field as it will change the structure of data in app insights.

All the events before this go live will have the field `agencyId` and then after go live it will be called `prisonId` which will make writing reporting queries difficult